spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Index page', type: :feature do
+  before(:each) do
+    @ingredient_1 = Ingredient.create(name: 'Rice', cost: 4)
+    @ingredient_2 = Ingredient.create(name: 'Chicken', cost: 7)
+    @ingredient_3 = Ingredient.create(name: 'Avocado', cost: 1)
+  end
+  
+  describe 'As a visitor' do
+    describe 'When I visit the ingredients index page' do
+      it 'I see the names of all ingredients listed' do
+
+        visit '/ingredients'
+
+        expect(page).to have_content(@ingredient_1.name)
```

Comment:
In this test we would also want to include the cost like you have it in your view
```ruby
expect(page).to have_content("#{@ingredient_1.name}: #{@ingredient_1.cost}")
```
I would also put that inside of a `within` block.